source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,192 @@
+
+Migrate Partition-Based Sync App to Flexible Sync
+-------------------------------------------------
+
+After beginning migration from Partition-Based Sync to Flexible Sync, the process
+is mostly automatic. However, you can control important parts of the migration.
+
+During a migration, you can cancel it. After a migration, you can roll it back 
+or manually commit the migration.
+
+Begin Migration
+~~~~~~~~~~~~~~~
+
+Before you begin a migration, make sure you're aware of the
+:ref:`migration stages <realm-sync-migrate-stages>` and
+:ref:`effects <realm-sync-migrate-what-to-know>`.
+
+During migration, client apps may experience side effects, most of which are similar
+to what happens if you disable and re-enable sync. These are some side effects
+of migration:
+
+- In the App Services UI, the Rules and Schemas pages will be unresponsive.
+- Your App Services Appp will experience a period of read-only sync and then a 
+  period of downtime.
+- If Partition-Based Sync default client max offline time is unset, the new
+  Flexible Sync config will set it to 30 days.
+- Storage in your Atlas cluster used by sync is temporarily doubled. Partition-Based
+  Sync and Flexible Sync must temporarily co-exist. Depending on the size of your
+  synced data, this could affect billing.
+
+.. QUESTION: If someone were to cancel a migration - or roll back a migration,
+   they'd potentially be on the hook for costs associated with subsequent 
+   migration attempts, correct?
+
+To migrate from Partition-Based Sync to Flexible Sync:
+
+#. In the App Services UI, go to your App and select the Sync page.
+#. At the top of the Sync page, select :guilabel:`[button-text]`.
+
+Your App will lose the migration in the following cases:
+
+- Partition-Based Sync is re-enabled
+- You roll back the migration
+
+Migrate permissions
+```````````````````
+
+Your App's sync permissions will be automatically migrated if the Partition-Based
+Sync permissions have direct Flexible Sync equivalents.
+
+Some Partition-Based Sync permission strategies can't translate directly to Flexible 
+Sync. You need to manually migrate permissions in these situations:
+
+- Use of ``%function``
+	- Can't be used in FBS document filters
+	- Most commonly used with the ``%partition`` expansion, which doesn't have an FBS equivalent
+
+- Use of a recursing operator (``%or``, ``%not``, ``%nor``, ``%and``)
+	- Technically works, but could cause issues.
+	- Users should test these and check if they need to make changes to the auto-migrated rules.
+	- Testing new permissions won't work on existing app. Need a new app to test manually-migrated permissions rules.
+
+Refer to the :ref:`Partiton-Based Sync <pbs-rule-migration>` reference for examples
+of how to map these Partition-Based Sync permissions to Flexible Sync permissions.
+
+Commit Migration
+~~~~~~~~~~~~~~~~
+
+After your migration has completed, it is in a non-permanent state. You can manually
```

Comment:
[nit] not a huge fan of "non-permanent" here since it may imply that we would roll it back without manual intervention

Reply:
Yeah, I don't think it's quite right. I've updated to `an evaluation state`. Do you think that conveys the purpose of the roll back window?